core/arch/instr.h

```diff
@@ -886,7 +886,8 @@ DR_API
 /**
  * Returns a copy of \p orig with separately allocated memory for
  * operands and raw bytes if they were present in \p orig.
- * Only a shallow copy of the \p note field is made.
+ * Only a shallow copy of the \p note field is made. The \p label_cb
+ * field will not be copied at all if \p orig is a label instruction.
  */
 instr_t *
```

Comment:
instr_set_label_callback needs documentation about this, to parallel the cloning note in instr_get_label_data_area

Reply:
Done.